examples/wasip3-http-axum-router/spin.toml

```diff
@@ -0,0 +1,20 @@
+#:schema https://schemas.spinframework.dev/spin/manifest-v2/latest.json
+
+spin_manifest_version = 2
+
+[application]
+name = "axum-router"
+version = "0.1.0"
+authors = ["Fermyon Engineering <engineering@fermyon.com>"]
```

Comment:
Nit: at some point, we should change these to "The Spin project" or whatever rubric we use.

(But even the Spin examples have this as `authors` so we can defer this to a later pass across all our repos.)